cypress/support/step_definition/room_boards/roomBoardSteps.spec.js

```diff
@@ -82,34 +83,26 @@ Then(
 );
 
 Then(
-	"I click on the button cancel in the video conference creation modal to back to the card",
+	"I click on the button cancel in the video conference creation modal to go back to the card",
 	() => {
 		roomBoards.clickCancelButtonInVideoConferenceCreationModal();
 	}
 );
 
-Then("I see multi-column copied board v-card on the room detail page", () => {
-	roomBoards.verifyMultiColumnCopiedBoardVCardVisibleOnRoomDetailPage();
+Then("I see copied multi-column board tile in the rooms details page", () => {
```

Comment:
you removed the hiven above between single and column, but not here and the other appearances, maybe make it consistent then

Reply:
I replaced it everywhere with hyphen (-) --> multi-column & single-column.